commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AbstractAuditLogSyslogReliableProtocolCommand.java

```diff
@@ -0,0 +1,105 @@
+package org.wildfly.extras.creaper.commands.auditlog;
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Batch;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+abstract class AbstractAuditLogSyslogReliableProtocolCommand extends AbstractAuditLogSyslogHandlerCommand {
```

Comment:
So, I'm not sure about this class. I'm not generally fond of deeper inheritance hierarchies, and the subclasses seem to repeat a lot of the code in this class (the `apply` methods, for example).